src/core/evaluator.js

```diff
@@ -2431,7 +2431,7 @@ class PartialEvaluator {
         return textContentItem;
       }
       const font = textState.font,
-        loadedName = font.loadedName;
+        loadedName = textState.loadedFontName;
```

Comment:
With the name-change mentioned elsewhere, the lines above could be simplified to:
```suggestion
      const { font, loadedName } = textState;
```